osf/management/commands/handle_duplicate_files.py

```diff
@@ -0,0 +1,262 @@
+import datetime
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+from osf.models import AbstractNode, Guid
+from addons.osfstorage.models import BaseFileNode
+
+logger = logging.getLogger(__name__)
+
+FOLDER = 'osf.osfstoragefolder'
+
+
+# For a specific file type, returns items that have duplicate
+# target_object_ids, names, parent_id, type, and _path
+# Basically, any items in same location with the same name.
+FETCH_DUPLICATES_BY_FILETYPE = """
+    SELECT *
+    FROM (
+      SELECT
+        target_object_id,
+        name,
+        parent_id,
+        type,
+        _path,
+        count(*) AS ct
+      FROM osf_basefilenode
+      WHERE type = %s
+      AND name != ''
+      GROUP BY (target_object_id, name, parent_id, type, _path)
+    ) as foo
+    WHERE ct > 1;
+    """
+
+# googledrive and trashedfile/folder/folders excluded.
+valid_file_types = [
+    'osf.onedrivefile',
+    'osf.gitlabfile',
+    'osf.dropboxfile',
+    'osf.githubfile',
+    'osf.s3file',
+    'osf.boxfile',
+    'osf.figsharefile',
+    'osf.osfstoragefile',
+    'osf.bitbucketfile',
+    'osf.owncloudfile',
+    FOLDER
+]
```

Comment:
So far, I'm only seeing 3 osfstoragefolders and 14 osfstoragefiles where the users need to be contacted.